pyproject.toml

```diff
@@ -1,6 +1,6 @@
 [tool.poetry]
 name = "great_expectations_cloud"
-version = "20240417.0"
```

Comment:
This update makes a lot of dependency changes.
- I would suggest waiting until #204 merges, making this a pre-release, and doing some manual testing before merging this.

Reply:
ok great idea @Kilo59. I'll follow up here tomorrow.

Reply:
Agree. @Kilo59 Would it make sense to revert the docker image bump PR as well in the meantime? I'm thinking yes. https://github.com/great-expectations/cloud/pull/205 Then incorporate that change with this in a pre-release as you suggested.